lms/templates/view.html

```diff
@@ -68,7 +68,7 @@ <h1>{{ _('תצוגת תרגיל') }} {{ solution['exercise']['id'] }}: {{ soluti
         </div>
       </div>
       <div id="code-view" data-id="{{ solution['id'] }}" data-file="{{ current_file.id }}" data-exercise="{{ solution['exercise']['id'] }}" data-role="{{ role }}" data-solver="{{ solution['solver']['fullname'] }}" data-allowed-comment="{{ config['USERS_COMMENTS'] | lower }}">
-        <pre><code id="user-code" class="language-{{ current_file | language_name }} line-numbers highlight">{% if current_file.path.rpartition('.')[-1] in image_extensions %}<div class="code-image-block"><img class="code-image" src="data:image/jpg;base64,{{ current_file.code }}"></div>{% else %}{{- current_file.code | trim(chars=' ') | e -}}{% endif %}</code></pre>
+        <pre><code id="user-code" class="language-{{ current_file | language_name }} line-numbers highlight">{% if current_file | language_name in image_extensions %}<div class="code-image-block">{% if current_file | language_name == 'svg' %}<img class="code-image" src="data:image/{{ current_file | language_name }}+xml;base64,{{ current_file.code }}">{% else %}<img class="code-image" src="data:image/{{ current_file | language_name }};base64,{{ current_file.code }}">{% endif %}</div>{% else %}{{- current_file.code | trim(chars=' ') | e -}}{% endif %}</code></pre>
```

Comment:
Please create a dictionary in utils.files to manage the mimetype; currently it's only semi-generic (for `.jpg`, for example, the mimetype is `img/jpeg` and not `img/jpg`, for `.ico` it's `x-icon` and so on). You can use [`mimetype.guess_type`](https://docs.python.org/3/library/mimetypes.html#mimetypes.guess_type).